testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/social/MicrosoftLoginPage.java

```diff
@@ -35,6 +35,12 @@ public class MicrosoftLoginPage extends AbstractSocialLoginPage {
     @FindBy(id = "idSIButton9")
     private WebElement submitButton;
 
+    @FindBy(xpath = "//input[contains(@class,'button_primary')]")
+    private WebElement staySignedIn;
+
+    @FindBy(xpath = "//input[contains(@class,'btn-primary')]")
```

Comment:
I have used xpath because I feel like it actually increases readability as it explicitly says what type of WebElement you are dealing with. I have also noticed that both approaches are used throughout the testsuite. WDYT ?

Reply:
It's just an opinion from my side and we don't have to strictly stick to it. I'm okay with xpaths overall, but I personally do not prefer them.